### PR TITLE
Minor cleanup

### PR DESCRIPTION
**Pull request by BrennanConroy**
Mostly wanted to fix the structured logging casing and saw a couple other minor things

### CODE REVIEW

File: src/Hosting/Systemd/src/SystemdLifetime.cs [hunk 1]
```diff
@@ -53,7 +53,7 @@ public Task WaitForStartAsync(CancellationToken cancellationToken)
 
         private void OnApplicationStarted()
         {
-            Logger.LogInformation("Application started. Hosting environment: {envName}; Content root path: {contentRoot}",
+            Logger.LogInformation("Application started. Hosting environment: {EnvName}; Content root path: {ContentRoot}",
```
Comment (davidfowl) [inline src/Hosting/Systemd/src/SystemdLifetime.cs]: Environment?

Comment (BrennanConroy) [reply src/Hosting/Systemd/src/SystemdLifetime.cs]: Prefer "EnvironmentName"? or "Hosting Environment:"?

Comment (davidfowl) [reply src/Hosting/Systemd/src/SystemdLifetime.cs]: {EnvironmentName}

File: src/Hosting/Systemd/src/ServiceState.cs [hunk 2]
```diff
@@ -35,7 +35,7 @@ public ServiceState(string state)
         /// String representation of service state.
         /// </summary>
         public override string ToString()
-            => _data == null ? string.Empty : Encoding.UTF8.GetString(_data);
```
Comment (Tratcher) [inline src/Hosting/Systemd/src/ServiceState.cs]: Can still be null using the struct default constructor.

Comment (BrennanConroy) [reply src/Hosting/Systemd/src/ServiceState.cs]: Smart guy

**Pull request merged**